src/servers/cron.ts

```diff
@@ -40,6 +41,11 @@ const main = async () => {
       const updatePendingOnChainPayments = () =>
         updateUsersPendingPayment({ onchainOnly: true })
 
+      const updateLnPaymentsCollection = async () => {
+        const result = await updateLnPayments()
+        if (result instanceof Error) throw result
```

Comment:
not sure we want to throw? it may stop the other promise to complete?

I think you can just pass `updateLnPayments` to the array of tasks below instead?

Reply:
Sure, I was following the precedent from [above](https://github.com/GaloyMoney/galoy/blob/8f7c0f0bb2f39ec955a6c7e37139f9c045857509/src/servers/cron.ts#L50).

@dolcalmi was there a reason why you used `throw` on Ln 51, or can we remove both of these?

Reply:
check line 82-83, we need to throw here because we need to catch and log the error, this will also affect exit code and will trigger an alert in slack (if we dont throw then we need to add additional logic to register the error and propagate the error)

Reply:
Ok so other promises are unaffected then? (just confirming)